instructor/models.py

```diff
@@ -217,6 +217,18 @@ class MicroscopyImages(models.Model):
     ('', 'Select Analysis'), ('Anti', 'Antibody-labeling'),
     ('Dye', 'Dye/Stain')
 )
+FACS_SCALE_TYPES = (('linear', 'Linear'), ('log', 'Logarithmic Scale'))
+
+
+class FlowCytometry(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='flow_cytometry')
+    scale = models.CharField(
+        max_length=50,
+        default='linear',
+        choices=FACS_SCALE_TYPES
+    )
+    xrange = models.IntegerField(default=200)
+    tick_values = models.CharField(max_length=50, default='50,100,150')
```

Comment:
I would define a couple of "constants" 
```
LINEAR = 'linear'
LOG = 'log'
```
and then define the tuple of tuples as
```
FACS_SCALE_TYPES = ((LINEAR, 'Linear'), (LOG, 'Logarithmic Scale'))
```
so that you can then assign to the default in this way:
```
scale = models.CharField(
    max_length=50,
    default=LINEAR,
    choices=FACS_SCALE_TYPES
)
```

It does not change the functionality, but improves readability